svm/Cargo.toml

```diff
@@ -75,7 +75,9 @@ solana-svm-transaction = { workspace = true }
 solana-system-interface = { workspace = true }
 solana-sysvar-id = { workspace = true }
 solana-timings = { workspace = true }
-solana-transaction-context = { workspace = true }
+
+[target.'cfg(debug_assertions)'.dependencies]
+solana-transaction-context = { workspace = true, features = ["debug-signature"] }
```

Comment:
I get that we only need `debug-signature` if `debug_assertions` are enabled, but this is the whole block:
https://github.com/anza-xyz/agave/blob/c60557c099be178b4dda1e6e91de2f92eb084049/svm/src/transaction_processor.rs#L852-L864

We always create a `TransactionContext`, so if we gate the entire dependency behind `debug_assertions`, then we're not explicitly pulling the crate in for the non-debug-assertions crate right ?

Reply:
Yes. The non-`debug_assertions` case still needs the dependency, but without the feature.